charts/super-agent-deployment/values.yaml

```diff
@@ -85,7 +85,7 @@ proxy: ""
 
 # -- (bool) Send the metrics to the staging backend. Requires a valid staging license key. Can be configured also with `global.nrStaging`
 # @default -- `false`
-nrStaging:
+nrStaging:  # TODO: Update OpAMP `endpoint` and auth `token_url` for staging and Europe when nrStaging is enabled.
```

Comment:
Nit: On top of adding the TODO, should we update the comment above to say that the other fields should be updated accordingly?